jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -51,6 +51,17 @@
         ],
       },
       replicas: 3,
+      labels: {
+        'app.kubernetes.io/name': 'alertmanager-' + $._config.alertmanager.name,
```

Comment:
what's the _config.aleretmanager.name?

Reply:
it is `main`. This will result in `app.kubernetes.io/name: alertmanager-main`, same as statefulset name

Reply:
I've always thought `name` is supposed to be the actual name of the "project", so I think it should just be "alertmanager" here, and `instance` is the concrete name of this instance